src/Microsoft.TestPlatform.Extensions.BlameDataCollector/ProcDumpArgsBuilder.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.TestPlatform.Extensions.BlameDataCollector
+{
+    using System.Collections.Generic;
+    using System.Text;
+
+    public class ProcDumpArgsBuilder : IProcDumpArgsBuilder
```

Comment:
The code is getting duplicated, you could use builder pattern to build these arguments. something like, procdumpargsbuilder.AddException().AddDumpType().Build()